adapter/internal/oasparser/envoyconf/constants.go

```diff
@@ -122,3 +125,9 @@ const (
 	// Azure tracer's name
 	TracerTypeAzure = "azure"
 )
+
+// Constants relevant to the rate-limit service
+const (
+	OperationLevelRateLimit     string = "OPERATION"
+	APILevelRateLimit           string = "API"
```

Comment:
Let's use RateLimit word first

For eg:
- RateLimitPolicyOperationLevel
- RateLimitPolicyAPILevel

Reply:
Can we also go without `Policy`?